.clang-format

```
@@ -0,0 +1,11 @@
+---
+BasedOnStyle: Google
+ColumnLimit: '100'
```

```suggestion
ColumnLimit: '120'
```

We can increase the max line length to be consistent with ROOT format:

https://github.com/root-project/root/blob/4389feb6d700c1d515014e56a3bcdee99cb6b4a5/.clang-format#L39

---

What about add aditional step in test? As far as I rememeber in FairRoot they have some scripts that perform additional check - if this check fail then you get an information.
However I have no Idea how root4star is tested.